test/Libraries/DynamoMSOfficeTests/DynamoMSOfficeTests.csproj

```diff
@@ -11,16 +11,20 @@
     <RootNamespace>DynamoMSOfficeTests</RootNamespace>
     <AssemblyName>DynamoMSOfficeTests</AssemblyName>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
 	  <PackageReference Include="NUnit" Version="2.6.3" />
-    <PackageReference Include="NUnitTestAdapter" Version="2.3.0" ExcludeAssets="all" />
-      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
-    <Reference Include="PresentationCore" />
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
+    <PackageReference Include="NUnitTestAdapter" Version="2.3.0" ExcludeAssets="all" /> 
   </ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' "> 
+        <PackageReference Include="NUnit" Version="3.13.3" ExcludeAssets="none" />
+        <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />   
+    </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\src\DynamoCoreWpf\DynamoCoreWpf.csproj">
+      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" ExcludeAssets="none" />
+      <Reference Include="PresentationCore" />
```

Comment:
these references to PresentationCore, System and SystemCore should not be in dotnet6.